Latex/inhalt/Abstract_Freigabeerklärung_Bachelorthesis.tex

```diff
@@ -0,0 +1,92 @@
+\hbadness=10000
+\cleardoublepage
+\section{Abstract zur Bachelorthesis}
+    % Alternativ zu Vektorgrafik-Logos (SVG) werden Befehle zur Nutzung von Pixelgrafik-Logos (z.B. PNG) bereitgestellt,
+    % falls Inkscape nicht installiert werden, bzw. nicht der PATH-Variable hinzugefügt werden soll.
+    %! Die SVG-Version sieht im Druck deutlich besser aus.
```

Comment:
ist der kommentar irgendwie doppelt da? kommt mir bekannt vor

Reply:
der kommt so auch in den Erklärung_X.texs vor

Reply:
whoops, stimmt